packages/react-components/src/components/FileUploadCards.tsx

```diff
@@ -47,7 +59,7 @@ export const _FileUploadCards = (props: FileUploadCardsProps): JSX.Element => {
               key={file.id}
               fileExtension={extension(file.filename)}
               actionIcon={
-                <IconButton className={iconButtonClassName}>
+                <IconButton className={iconButtonClassName} ariaLabel={localeStrings.removeFile}>
```

Comment:
Do we need the filename in aria label here as well similar to filedownloadcard?
If not, can we remove it from filedownloadcard as well?
Ideally, filecard should announce the filename when someone tabs on it.

Reply:
done